mobile/src/reducers/calendar/event-dialog/event-dialog.model.ts

```diff
@@ -56,6 +64,39 @@ export class EventDialogEmptyModel extends EventDialogModel<EventDialogEmptyMode
 }
 
 export class ClaimSickLeaveDialogModel extends EventDialogModel<ClaimSickLeaveDialogModel> {
+    public readonly title = 'Select date to Start your Sick Leave';
+    public readonly icon = 'sick_leave';
+    public readonly cancelLabel = 'Back';
+    public readonly acceptLabel = 'Confirm';
+
+    public startDate: Moment;
```

Comment:
should this be publicly accessable for modification?

Reply:
Yes, it should. The idea was to have an ability to clone dialog and change this field in reducers. But _outside_ it is used only as EventDialogModel base type, so no one can access to this field.

Reply:
why not make a method 'withStartDate(date: Moment) { const instance = this.clone(); instance.startDate = date; return instance; } instead? Otherwise someone will change it.